tests/test_quicksort.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from big_o import big_o, complexities, datagen
+
+from src.quicksort import quicksort
+
+
+class TestQuicksort(unittest.TestCase):
+    def test_sort(self):
+        self.assertEqual(quicksort([3, 2, 5, 1, 4]), [1, 2, 3, 4, 5])
```

Comment:
test quicksort([])

Reply:
Why?

Reply:
Because any man that implements function can forget to check length of array. We know you didn't, but if we mean that tests is written independently of implementation they check, it's more proper to aim for checking all cases